### PR TITLE
Contact form & translations

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/CMS-59

### CODE REVIEW

File: contact/forms.py [hunk 1]
```diff
@@ -300,66 +295,15 @@
 
 
 class ContactForm(forms.Form):
-    def __init__(self, *args, **kwargs):
-        self.helper = FormHelper()
-        self.helper.layout = Layout(
```
Comment (richtier) [inline contact/forms.py]: :heart: much simpler

File: contact/forms.py [hunk 2]
```diff
@@ -369,15 +313,24 @@ def __init__(self, *args, **kwargs):
     )
     country = fields.ChoiceField(
         label=_('Which country are you based in?'),
-        choices=COUNTRIES
-
+        help_text=_(
+                'We will use this information to put you in touch with '
```
Comment (richtier) [inline contact/forms.py]: bit too indented here.

File: contact/templates/contact/contact.html [hunk 3]
```diff
@@ -1,23 +1,104 @@
...
+    <div class="form-group{% if form.name.errors %} form-group-error{% endif %}">
+      {{ form.name.label_tag }}
+      <span class="error-message">{{ form.name.errors }}</span>
+      {{ form.name }}
+    </div>
+    <div class="form-group{% if form.job_title.errors %} form-group-error{% endif %}">
+      {{ form.job_title.label_tag }}
+      <span class="error-message">{{ form.job_title.errors }}</span>
+      {{ form.job_title }}
+    </div>
```
Comment (richtier) [inline contact/templates/contact/contact.html]: there looks like a bit of repetition in the template. could do

```
# thing.html
    <div class="form-group{% if field.errors %} form-group-error{% endif %}">
      {{ field.label_tag }}
      <span class="error-message">{{ field.errors }}</span>
      {{ field }}
    </div>
```

and then

```
# contact.html
{% include 'thing.html` with field=form.job_title %}
{% include 'thing.html` with field=email.job_title %}
```

or even

```
{% for field in form %}
    {% include 'thing.html` with field=field %}
{% endfor %}

```

File: locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -161,14 +167,56 @@ msgstr ""
...
 
+#: core/templates/core/includes/header.html:31
+msgid "Menu"
+msgstr ""
+
 #: core/templates/core/includes/header_language_select.html:7
 msgid "Select language"
 msgstr ""
+
+#, fuzzy
```
Comment (richtier) [inline locale/es/LC_MESSAGES/django.po]: general FYI about `#, fuzzy` - if you notice translations not working, look in the .po file and check if the translation has been marked as `#, fuzzy` - that would kill the translation.

File: contact/templates/contact/contact.html (hunk 3 above)
Comment (eadpearce) [reply contact/templates/contact/contact.html]: oh i was wondering how i could simplify this. neat solution

File: locale/es/LC_MESSAGES/django.po (hunk 4 above)
Comment (eadpearce) [reply locale/es/LC_MESSAGES/django.po]: yep i noticed that before. these seem to be okay though

**Pull request merged**